mindjunkies/courses/forms.py

```
@@ -2,47 +2,50 @@
+            "preview_video": forms.FileInput(),
         }
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         for field in self.fields.values():
-            field.widget.attrs.update({
-                'class': 'w-full rounded-md border-gray-300 shadow-sm focus:border-indigo-300 focus:ring focus:ring-indigo-200 focus:ring-opacity-50'
-            })
+            field.widget.attrs.update(
+                {
+                    "class": "w-full rounded-md border-gray-300 shadow-sm focus:border-indigo-300 focus:ring focus:ring-indigo-200 focus:ring-opacity-50"
```

Ruff (0.8.2)

Line too long (153 > 119)

(E501)